tomviz/RAWFileReaderDialog.cxx

```diff
@@ -0,0 +1,153 @@
+/******************************************************************************
+
+  This source file is part of the tomviz project.
+
+  Copyright Kitware, Inc.
+
+  This source code is released under the New BSD License, (the "License").
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+******************************************************************************/
+#include "RAWFileReaderDialog.h"
+#include "ui_RAWFileReaderDialog.h"
+
+#include "vtkSMPropertyHelper.h"
+#include "vtkSMProxy.h"
+
+#include <QComboBox>
+#include <QPushButton>
+#include <QSpinBox>
+#include <QtGlobal>
+
+namespace tomviz {
+
+RAWFileReaderDialog::RAWFileReaderDialog(vtkSMProxy* reader, size_t filesize,
+                                         QWidget* p)
+  : QDialog(p), m_ui(new Ui::RAWFileReaderDialog), m_reader(reader),
+    m_filesize(filesize)
+{
+  m_ui->setupUi(this);
+  connect(m_ui->dimensionX,
+          static_cast<void (QSpinBox::*)(int)>(&QSpinBox::valueChanged), this,
```

Comment:
I would have liked to use `qOverride` here since it is less ugly, but I couldn't get the compiler to find the symbol, even with the `<QtGlobals>` header included.